packages/sdk-auth/src/auth.js

```diff
@@ -106,11 +107,28 @@ export default class SdkAuth {
       config.token || SdkAuth._encodeClientCredentials(credentials)
     const authType = config.authType || constants.DEFAULT_AUTH_TYPE
 
-    let body = `grant_type=${grantType}`
-    if (grantType !== 'refresh_token') body += `&scope=${scope}`
-    if (disableRefreshToken === true) body += '&refresh_token=false'
+    const isNotRefreshTokenGrantType = grantType !== 'refresh_token'
+    const initialBody = encode({
+      grant_type: grantType,
+      ...(disableRefreshToken && { refresh_token: false }),
+      ...(isNotRefreshTokenGrantType && { scope }),
+    })
 
-    return { basicAuth, authType, uri, body, headers }
+    return { basicAuth, authType, uri, body: initialBody, headers }
+  }
+
+  static _appendToRequestBody(
+    request: AuthRequest,
+    toAppend: Object
+  ): AuthRequest {
+    const previousDecodedRequestBody = request.body ? decode(request.body) : {}
+    const nextEncdedRequestBody = encode({
```

Comment:
nitpick typo:
~~nextEncdedRequestBody~~
nextEnc**o**dedRequestBody

Reply:
Thanks. Feel free to use GitHub's review suggestions next time. Helps to get better diffs of what you carefully markdowned yerself :)